app/src/main/java/com/droidhats/campuscompass/viewmodels/SplashViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package com.droidhats.campuscompass.viewmodels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class SplashViewModel : ViewModel() {
+
+    //set as false to identify instantiations are not complete
+    private val mutableLiveData = MutableLiveData<Boolean>().apply {
+        value = false
+    }
+
+    //example of a value that can be accessed from SplashFragment
+    val hardCode: LiveData<Boolean> = mutableLiveData
+
+    fun init() {
+        mutableLiveData.value = true //hard coded to show successful API connection
+        returnData() //dummy function - represents Map initialization methods
```

Comment:
I don't think we need a dummy function here, especially if it doesn't serve any purpose